_posts/2021-07-01-pulpcore-3.14-release-announcement.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Pulp 3.14 is now Generally Available
+author: Melanie Corr
+tags:
+  - release
+---
+
+# 3.14 - headline features
+
+We are happy to announce the release of Pulpcore 3.14!
+
+This release contains a number of bug fixes and enhancements. In this blog, we'll take a look at
+the headline features.
+
+A gentle reminder that the [Community Survey](https://forms.gle/KJDoSdqWf5ajut5f6) is open,
+and we would love to hear from you.
+
+## Pulp has a new queueless tasking system! [#8501](https://pulp.plan.io/issues/8501)  
+
+The largest and most significant change introduced with this release is the new tasking system.
+ With this change, Pulp no longer needs a resource manager.
+
+If you'd like an in-depth look at the motivations, benefits, and details of the new tasking system,
+read [Matthias Dellweg's blog post](https://pulpproject.org/2021/06/21/a-new-tale/) and also watch
+ a YouTube livestreamed interview with Matthias conducted by Pulp's Brian Bouterse:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YWKw4RYluPM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Note that for the moment, this feature is in tech-preview.
+Please [let us know what you think](https://pulpproject.org/help/#github-discussions).
+
+### Sticking with the old tasking system?
+If you intend to keep using the old tasking system, you should configure the `USE_NEW_WORKER_TYPE`
+setting to `false` before upgrading. For more information about how to enable or disable the new
```

Comment:
```suggestion
setting to `False` before upgrading. For more information about how to enable or disable the new
```

I know, this is weird. But `USE_NEW_WORKER_TYPE` looks like python and falsity should be stated accordingly.
If it were Ansible (and yaml therin), `settings.use_new_worker_type: false` were preferred. Luckily YAML also accepts capitalized `False`.